src/js/web-components/video-player/VideoPlayer.js

```diff
@@ -345,6 +353,9 @@ export default class extends HTMLElement {
 
     this.videoElement.addEventListener('loadedmetadata', setPipButton);
     this.videoElement.addEventListener('emptied', setPipButton);
+    this.videoElement.addEventListener('enterpictureinpicture', () => this.classList.add(PIP_CLASSNAME));
```

Comment:
Nit: It would be great if the PiP floating button would also change shape when video is playing in picture in picture.

Reply:
@beaufortfrancois What do you mean by "change shape"? Like change the icon from the current "Enter PiP" icon to some kind of "Exit PiP" icon?

Reply:
Yep!

Reply:
@beaufortfrancois Gotcha! Added in e484469. Here's a short screencast:

https://user-images.githubusercontent.com/433570/148036073-6499e956-742f-45dd-bfe8-8d8535053018.mp4

(This one's on me, our designer has originally designed both icons, but I missed it during implementation. 😐)

Reply:
LGTM. Thank you @dero! See https://github.com/GoogleChrome/kino/pull/171#commitcomment-62881879 as well